### ISSUE TITLE
Accessibility for blind users

### ISSUE THREAD

**Issue by pzajda** [ricochet-im/ricochet#382]
Hi,

I use Ricochet on Windows with NVDA as screen reader.
Unfortunatly, a lot of things are not usable for blind users.
First of all, the contact list is not visible, it seems keyboard navigation doesn't allow to focus the contact list. Only preferences abutton and the one to change the status are focusable.
Another annoying problem exists, we cannot read the message list at all.
Would it be possible to implement better QT Accessibility to Ricochet to make it usable even for blind users please?
I'm of course available if you need more information.
Thanks for your great work.

**Comment by meZee** [comment]
Seconded.
